.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+### Bug Report
```

Comment:
Yeah, unfortunately Github doesn't have a great way to split incoming issues into bugs vs. feature requests. I've used this template on TSLint for a little while and find that it generally works -- bug reports benefit from boilerplate, while feature requests don't as much. For now, the standard operating procedure for contributors will be to delete this template when trying to file an FR (I'll add that in an HTML comment in a follow-up).